docs/ldml/tr35-keyboards.md

```diff
@@ -10,6 +10,12 @@
 
 For the full header, summary, and status, see [Part 1: Core](tr35.md).
 
+#### _Important Note_
+
+> The CLDR [Keyboard Workgroup](https://cldr.unicode.org/index/keyboard-workgroup) is currently
+> developing major changes to the CLDR keyboard specification. These changes are targetted for

```

Comment:
```suggestion
> developing major changes to the CLDR keyboard specification. These changes are targeted for
```

"targetted" is not technically illegal but it is certainly unusual...

"Major" or "significant"?  But we can nit pick forever.

Looks Good To Me.

Reply:
thanks. @mcdurdin changed it to targeted